docs/src/content/docs/guides/migrating-from-terraform-cloud.mdx

```diff
@@ -0,0 +1,207 @@
+
+## Pre-Migration Planning
+
+A successful migration starts with a thorough plan. Begin by taking an inventory of your TFC setup:
+
+<Steps>
+1. List Organizations and Workspaces  
+   - Note each workspace's purpose (project, environment, etc.) and the Terraform code repository/directory it connects to.  
+   - Record the Terraform version and any relevant workspace settings.
+
+2. State Location and Names  
+   - TFC stores state within each workspace. Note each workspace name, since it often forms part of the key path if you use an S3-like backend after migration.
+
+3. Variables and Secrets  
+   - Document all TFC workspace variables, both sensitive and non-sensitive. Sensitive values can't be viewed directly in TFC if you never saved them elsewhere, so plan to regenerate or locate them securely.
+
+4. Workflow Triggers  
+   - Understand how runs are triggered (automatic via VCS push, manual in UI, CLI-driven, etc.). If you rely on TFC's run tasks or webhooks, decide how you'll replicate them with Terrateam.
+
+5. Compliance/Approvals  
+   - Check if TFC used Sentinel, manual approvals, or RBAC. Decide how you'll replicate or improve those controls in Terrateam using GitHub reviews, built-in Terrateam features like Access Controls or Apply Requirements, OPA, or other policy tools.
+
+6. Choose a New Backend  
+   - Terrateam doesn't host your state. Common choices include AWS S3 (+ DynamoDB for locking), Azure Blob, GCS, or another remote backend. Create the backend, ensuring versioning/encryption for compliance.
+
+7. Security, Governance, and Secrets  
+   - Evaluate how you'll store secrets going forward (e.g. GitHub Secrets). Review how you'll replicate TFC's role/team restrictions, usually via GitHub team permissions and `.terrateam/config.yml` access controls.
+</Steps>
+
+Compile these into a migration checklist for each workspace so nothing (like a crucial API key) is overlooked.
+
+---
+
+## Migrating Terraform State to S3
+
+Below is an example migration to AWS S3. If you use other cloud backends, adapt accordingly:
+
+<Steps>
+1. Export State from TFC  
+     ```bash
+     terraform login            # authenticate with TFC
+     terraform state pull > state.tfstate
+     ```  
+     This pulls the current TFC state into a local file.  
+
+2. Set Up S3  
+   - Create an S3 bucket (e.g., `mycompany-terraform-state`) with versioning and encryption.  
+
+3. Update Terraform Backend  
+   - In your Terraform config (`backend.tf` or within `terraform { backend "s3" { ... } }`):
+     ```hcl
+     terraform {
+       backend "s3" {
+         bucket         = "mycompany-terraform-state"
+         key            = "prod/network/terraform.tfstate"
+         region         = "us-west-2"
+         dynamodb_table = "terraform-state-locks"
+         encrypt        = true
+       }
+     }
+     ```
+
+4. Migrate the State  
```

Comment:
I don't understand this step.  Why do I comment out the backend then enable it again?  I don't run any commends between those two steps.